shared/controller-api/src/main/proto/Controller.proto

```diff
@@ -279,6 +279,7 @@ message ContinuationToken {
 message StreamInfo {
     string scope = 1;
     string stream = 2;
+    string requestedPermission = 3;
```

Comment:
Would using an enum simplify the code further?

Reply:
Yes, that makes sense. Will do so.

Reply:
I've just pushed changes for modifying the datatype of this field to enum. I've also renamed it to `AccessOperation`, to better represent it's intent.